cypress/support/utils/product.js

```diff
@@ -179,16 +179,12 @@ export class Product {
     cy.selectTab('M_ProductPrice');
     cy.pressAddNewButton();
     cy.writeIntoLookupListField('M_PriceList_Version_ID', productPrice.priceList, productPrice.priceList, true, true);
-
-    cy.writeIntoStringField('PriceList', productPrice.listPriceAmount, true, null, true);
-    cy.writeIntoStringField(
-      'PriceStd',
-      productPrice.standardPriceAmount,
-      true /*modal*/,
-      null /*rewriteUrl*/,
-      true /*noRequest*/
-    );
-    cy.writeIntoStringField('PriceLimit', productPrice.limitPriceAmount, true, null, true);
+    if(productPrice.listPriceAmount != 0)
```

Comment:
Maybe it's just me, but i want each `if` to have curly braces around the code it handles, even if only a single line follows. I feel it's safer for whoever may edit next, plus depending on the ide (or notepad) used, stuff may not be aligned properly, and the future editor may add another line believing that it belongs to the if, even though it doesn't.

Reply:
Second q for all these edits: why do we have to check ` if(productPrice.standardPriceAmount != 0) { }` instead of just ` if(productPrice.standardPriceAmount) { }`? 

Is "productPrice=0" an edge case which gives errors from backend (or doesn't show an error, but it should?

Reply:
Not an edge case, I just thought it would make it faster *clownface*